streamwebs_frontend/streamwebs/tests/views/test_registration_view.py

```diff
@@ -1,7 +1,11 @@
-from django.test import Client, TestCase
+from django.test import Client, TestCase, override_settings
 from django.core.urlresolvers import reverse
 
 
+@override_settings(SCHOOL_CHOICES=(
+    ('a', 'School A'),
```

Comment:
We found out that this decorator/overriding settings doesn't seem to work on these tests (maybe because it doesn't apply to post requests/forms?). As of the develop branch, there is a currently a default school 'a' in settings.py so that tests can pass, so it's okay if you delete this decorator.